src/lib/executeQuery.ts

```diff
@@ -23,10 +23,17 @@ const queryToOdataString = (query): string => {
   return result;
 };
 
-const processIncludes = (queryBuilder: any, odataQuery: any, alias: string) => {
+const processIncludes = (queryBuilder: any, odataQuery: any, alias: string, parent_metadata: any): [any, string] =>  {
```

Comment:
Technically, the metadata here isn't used yet, but it may be helpful in the future to have access to these properties. (for example, for the todo: `// todo: remove columns that are isSelect: false`).